cmd/aws-janitor/main.go

```diff
@@ -162,7 +165,15 @@ func markAndSweep(opts resources.Options, region string) error {
 	if err != nil {
 		return errors.Wrapf(err, "-path %q isn't a valid S3 path", *path)
 	}
-
+	if *enableStateBucketCheck {
```

Comment:
A state bucket must be tagged with exclude-tags, otherwise, we just return error to avoid deleting it.

Reply:
Great idea! Probably can be code comment too.